indico/modules/events/surveys/blueprint.py

```diff
@@ -45,6 +46,11 @@
                  methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/surveys/<int:survey_id>/open', 'open_survey', RHOpenSurvey, methods=('POST',))
 _bp.add_url_rule('/manage/surveys/<int:survey_id>/close', 'close_survey', RHCloseSurvey, methods=('POST',))
+_bp.add_url_rule('/manage/surveys/<int:survey_id>/export-submissions', 'export_submissions', RHExportSubmissions)
```

Comment:
We don't support selecting which submissions to export? (imho we don't need it, but we had it before. check with @pferreir )

Because if we need to support that, this endpoint should be POST-only

Oh, and I'd use 'submissions.csv' instead of 'export-submissions' for the URL.